tests/notebooks/e2e-wine/e2e-wine-kfp-mlflow-seldon.ipynb

```diff
@@ -0,0 +1,1233 @@
+    },
+    "tags": [
+     "raises-exception"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "assert run.data.tags.get(\"author\") == \"kf-testing\"\n",
+    "assert_mlflow_run_finished(mlflow, run.info.run_id)\n",
+    "assert_mlflow_model_exists(mlflow, run.info.run_id)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "id": "a2606489-1dc6-47c1-adc0-d8e706cfbc8c",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2023-08-09T14:52:51.876437Z",
+     "iopub.status.busy": "2023-08-09T14:52:51.876126Z",
+     "iopub.status.idle": "2023-08-09T14:52:51.947490Z",
+     "shell.execute_reply": "2023-08-09T14:52:51.937517Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# workflow component\n",
+    "training_op = kfp.components.create_component_from_func(\n",
+    "    func=training,\n",
+    "    output_component_file=\"train-component.yaml\",\n",
+    "    base_image=\"python:3.8.10\",\n",
+    "    packages_to_install=[\"boto3\", \"mlflow==2.1.1\", \"numpy\", \"pandas\", \"pyarrow\", \"scikit-learn<1.2\"],\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c10c6991",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Deploy Model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "id": "9599d635-f945-476f-a275-ec89eb5b080f",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2023-08-09T14:52:51.960956Z",
+     "iopub.status.busy": "2023-08-09T14:52:51.960011Z",
+     "iopub.status.idle": "2023-08-09T14:52:51.965410Z",
+     "shell.execute_reply": "2023-08-09T14:52:51.964458Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "SELDON_DEPLOYMENT_NAME = \"kf-testing\"\n",
+    "MODEL_NAME = \"wine-model\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "id": "86bdbfa6-fb71-4c7c-87e5-d2193bbd75c8",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2023-08-09T14:52:51.969361Z",
+     "iopub.status.busy": "2023-08-09T14:52:51.969032Z",
+     "iopub.status.idle": "2023-08-09T14:52:52.018048Z",
+     "shell.execute_reply": "2023-08-09T14:52:52.017130Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "def deploy(\n",
+    "    seldon_deployment_name: str = \"default_seldon_deployment_name\",\n",
+    "    model_uri: str = \"default_model_uri\",\n",
+    "    model_name: str = \"default_model_name\",\n",
+    "):\n",
+    "    import yaml\n",
+    "\n",
+    "    from kubernetes import client, config\n",
+    "\n",
+    "    manifest = \"\"\"\n",
+    "apiVersion: machinelearning.seldon.io/v1\n",
+    "kind: SeldonDeployment\n",
+    "metadata:\n",
+    "  name: \"\"\" + seldon_deployment_name + \"\"\"\n",
+    "spec:\n",
+    "  name: wines\n",
+    "  predictors:\n",
+    "  - componentSpecs:\n",
+    "    - spec:\n",
+    "        containers:\n",
+    "        - name: classifier\n",
+    "          image: seldonio/mlflowserver:1.17.0\n",
```

Comment:
Same comment as before about the version.

Reply:
Yeah, this one could be provided as an argument to the KFP run.